pkg/server/router.go

```diff
@@ -287,7 +288,11 @@ func Listen(config *conf.Config, serverWaitGroup *sync.WaitGroup, enterpriseSyst
 				serverWaitGroup.Add(1)
 				servers[key] = srv
 				go func() {
-					logging.Log.Error("", srv.ListenAndServe())
+					listenErr := srv.ListenAndServe()
+					logging.Log.Error("", listenErr)
+					if (*config).PanicOnBindError && strings.Contains(listenErr.Error(), "bind: address already in use") {
```

Comment:
Jeez. This is how we have to check for bind error? A string comparison?

Reply:
go is bad with errors lol